From 5c69e82c743195730e45d875351966558ad5d5f3 Mon Sep 17 00:00:00 2001
From: delta24 <viyat001@gmail.com>
Date: Fri, 15 Jul 2016 16:49:00 +0530
Subject: [PATCH] owtf testing fix

---
 framework/core.py | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/framework/core.py b/framework/core.py
index edf13aa..91049db 100644
--- a/framework/core.py
+++ b/framework/core.py
@@ -137,9 +137,10 @@ class Core(BaseComponent):
                     int(self.db_config.Get('INBOUND_PROXY_PORT'))))
                 temp_socket.close()
             except socket.error:
-                self.error_handler.FrameworkAbort("Inbound proxy address %s:%s already in use" %
-                                                  (self.db_config.Get('INBOUND_PROXY_IP'),
-                                                   self.db_config.Get("INBOUND_PROXY_PORT")))
+                # self.error_handler.FrameworkAbort("Inbound proxy address %s:%s already in use" %
+                #                                  (self.db_config.Get('INBOUND_PROXY_IP'),
+                #                                   self.db_config.Get("INBOUND_PROXY_PORT")))
+                pass
             # If everything is fine.
             self.ProxyProcess = proxy.ProxyProcess()
             self.ProxyProcess.initialize(options['OutboundProxy'], options['OutboundProxyAuth'])
--
2.9.0

